crates/rolldown_common/src/module/normal_module.rs

```diff
@@ -356,3 +355,18 @@ impl DerefMut for NormalModule {
 pub enum ModuleRenderArgs<'any> {
   Ecma { ast: &'any EcmaAst },
 }
+
+#[derive(Debug, Clone, Copy)]
+pub enum EsmNamespaceKind {
```

Comment:
Could you keep it as `bool`?

Reply:
> Could you keep it as `bool`?

Nice point! I can refactor this part to avoid over-engineering. Thanks for pointing it out!